Base/res/js/Spreadsheet/runtime.js

```diff
@@ -260,19 +261,28 @@ function range(start, end, columnStep, rowStep) {
 
 function R(fmt, ...args) {
     if (args.length !== 0) throw new TypeError("R`` format must be a literal");
-
-    fmt = fmt[0];
-
-    // CellName (: (CellName|ColumnName) (: Integer (: Integer)?)?)?
-    // ColumnName (: ColumnName (: Integer (: Integer)?)?)?
```

Comment:
Let's keep this around, the logic is still not as descriptive as this comment.
(Please update this if you've changed the syntax)

Reply:
Added the line a little lower with the updated syntax, let me know if that's not good enough

Reply:
That's fine, but it's not actually correct, if I'm reading the code right, it should be something like `ColRow:Col(Row)?(:ColStep(:RowStep)?)?`

Other than that, looks fine.

Reply:
Yeah, that's correct, this only allows 2 elements in the split array or 4: https://github.com/SerenityOS/serenity/pull/12692/files#diff-fe9c34a7f1460d29bd66e57cb638eabecb1e1836a62603040ade9b0c7b0d69c0R248 , as in ColRow:ColRow or ColRow:ColRow:ColStep:RowStep

Reply:
ah right, I missed that. alright, looks good.